ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java

```diff
@@ -564,7 +564,7 @@ public void run() {
               .put("output", initialMessageFrame.getOutputData().toHexString())
               .put("gasUsed", "0x" + Long.toHexString(evmGas))
               .put("pass", initialMessageFrame.getExceptionalHaltReason().isEmpty())
-              .put("fork", protocolSpec.getName());
+              .put("fork", protocolSpec.getHardforkId().description());
```

Comment:
Is this String identical to the former getName() or a potential breaking change?

Reply:
yes, I changed the description of the `HardforkId` enum where needed to match the string returned by the name method